packages/ckeditor5-table/src/ui/colorinputview.js

```diff
@@ -305,3 +306,16 @@ export default class ColorInputView extends View {
 		}
 	}
 }
+
+// Normalizes color value, by stripping extensive whitespace.
+// For example., transforms:
+// * `   rgb(  25 50    0 )` to `rgb(25 50 0)`,
+// * "\t  rgb(  25 ,  50,0 )		" to `rgb(25 50 0)`.
+//
+// @param {String} colorString The value to be normalized.
+// @returns {String}
+function normalizeColor( colorString ) {
+	// Remove any whitespace at the beginning, after `(`, or `,`, at the end, before `)`, `,`, or another whitespace.
+	// Then, replace `,` or whitespace with single space.
+	return colorString.replace( /(?<=^|\(|,)\s*|\s*(?=$|\)|\s|,)|/g, '' ).replace( /,|\s/g, ' ' );
```

Comment:
Unfortunatelly, fancy ECMA2018 RegExp additions are not supported on Firefox. It blows the editor up.

Reply:
We could either have a simpler RegExp for FF or have totally new different RegExp.

Reply:
ps.: This RegExp looks very complicated - couldn't we use something simpler (two-step change, like congesting multiple whitespaces to one space and then replacing quirks?

:point\_up: just a random thought as I didn't dig into the RegExp istelf as it cause FF to not load the editor in the first place.

Reply:
If you go with two RegExps, here's a similar case: https://github.com/ckeditor/ckeditor5/blob/dca2d0f620e0fc2895ebb56f57b5d00950ea55ef/packages/ckeditor5-mention/src/mentionui.js#L639-L655.

But I'd check if we could simplify that.

Reply:
It's the lack of support for lookbehind https://bugzilla.mozilla.org/show_bug.cgi?id=1225665 in FF. I'd do separate `.replace` for this assertion.

Reply:
>  couldn't we use something simpler (two-step change, like congesting multiple whitespaces to one space and then replacing quirks

That's exactly, what I do, see the code comment above.
https://github.com/ckeditor/ckeditor5/pull/6801/files/4ea684d2c61271b4d8a8f6f85239a7b6d390bb1c#diff-05504865ae0c3ced8720bd0b06af5c54R318-R319

Reply:
Let me know if it get clearer after https://github.com/ckeditor/ckeditor5/pull/6801/commits/d3513944db643ece09a531abafe479fb38a3a16e